.goreleaser.yaml

```diff
@@ -0,0 +1,89 @@
+# For more information on configuring GoReleaser, refer to the documentation:
+# https://goreleaser.com
+#
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+
+archives:
+  - format: tar.gz
+    # this name template makes the OS and Arch compatible with the results of `uname`.
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    # use zip for windows archives
+    format_overrides:
+      - goos: windows
+        format: zip
+
+nfpms:
+  - vendor: Datum Technology, Inc
+    homepage: https://www.datum.net/
+    maintainer: support@datum.net
+    license: Apache 2.0
+    provides:
+    - datumctl
+    recommends:
+    - kubectl
+    formats:
+      - apk
+      - deb
+      - rpm
+      - termux.deb
+      - archlinux
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+      - "^chore:"
+
+release: {}
+
+sboms:
+  - artifacts: archive
+
+brews:
+- name: cli
```

Comment:
Should we have this match the tool's name, `datumctl`?

Reply:
I used that originally but the brew command to install the tool would be `brew install datum-cloud/tap/datumctl` and I didn't like the stuttering. Using `cli` as the name here makes the install command `brew install datum-cloud/tap/cli`.

Reply:
I'm not super familiar with brew, but how does one get to the point of `brew install datumctl`? Does this naming choice introduce any barriers there?

I noticed that Pulumi uses `pulumi/tap/pulumi`, but GitHub's CLI is `brew install gh`.

Reply:
The format `pulumi/tap/pulumi` automatically discovers the brew formula to install the package. The format looks for a formula named `pulumi` in the repo `pulumi/hombrew-tap` (e.g. [pulumi tap formula](https://github.com/pulumi/homebrew-tap/blob/main/Formula/pulumi.rb)). To do `brew install gh` you have to make a PR with a formula to the hombrew core repo (e.g. [GitHub formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/g/gh.rb)).

Reply:
Looks like if you use `brew tap <user>/<repo>` that naming conflicts would require you install with a fully qualified name, per https://docs.brew.sh/Taps#formula-with-duplicate-names. I'm not sure what other brew packages we may have in the future, but having a name like `cli` may make adding the tap ahead of time pointless.

I'm not sure how often users do that though, so until we're closer to a v1 here I think I'm ok with `cli` for now.